src/symbolize/gimli/libs_aix.rs

```diff
@@ -26,15 +35,14 @@ pub(super) fn native_libraries() -> Vec<Library> {
             {
                 break;
             } else {
-                match Error::last_os_error().raw_os_error() {
-                    Some(libc::ENOMEM) => {
+                match errno() {
+                    libc::ENOMEM => {
```

Comment:
Not really sure if I should keep this change. If you want I will drop this commit.